docs/docs/administration/customize.md

```diff
@@ -1495,9 +1495,9 @@ Useful tools for development:
 | Parameter                           | Description                                                                                                     | Default value |
 | ----------------------------------- | --------------------------------------------------------------------------------------------------------------- | ------------- |
 | `userdata-bbb_multi_user_pen_only=` | If set to `true`, only the pen tool will be available to non-participants when multi-user whiteboard is enabled | `false`       |
-| `userdata-bbb_presenter_tools=`     | Pass in an array of permitted tools from `settings.yml`                                                         | all enabled   |
-| `userdata-bbb_multi_user_tools=`    | Pass in an array of permitted tools for non-presenters from `settings.yml`                                      | all enabled   |
-
+| `userdata-bbb_presenter_tools=`     | Pass in an array of permitted tools from `settings.yml`. The options we support are: `select, hand, draw, eraser, arrow, text, note, rectangle and *more`.                                                         | all enabled   |
```

Comment:
Could you please also include an example of how we can allow only the tools  `eraser` and `diamond`?
You have `rectangle` mentioned twice.
Could you split them in backticks individually: `rectangle`, `ellipse`, ...
Let's use copy-paste here (so both lines include the full list) of the detailed info rather than `*more` as it can be confusing.